src/sentry/integrations/vsts/client.py

```diff
@@ -149,11 +159,8 @@ def get_work_item_states(self, instance, project):
             api_preview=True,
         )
 
-    def get_work_item_types(self, instance, process_id):
-        return self.get(
-            VstsApiPath.work_item_types.format(instance=instance, process_id=process_id),
-            api_preview=True,
-        )
+    def get_work_item_categories(self, instance, project):
```

Comment:
I thought we didn't want the categories? we did want the types?

Reply:
@MeredithAnya We use the category API and then pull the work item types from the response